gdcdictionary/schemas/virus_genome_variation.yaml

```diff
@@ -39,6 +39,12 @@ links:
         target_type: virus_sequence_run_taxonomy
         multiplicity: many_to_many
         required: false
+      - name: virus_sequences
```

Comment:
did you forget to remove this link?

Reply:
I keep the links between `virus_genome_variation` (vcf file node) and `virus_sequence` because VCF is not part of the virus contig/blast analysis. VCF is more commonly used, so i don't want to make it available only in the genomic_data_file page. Although users can use that `genomic_data` tab to filter accession numbers, it's not enough for VCF, it will be better to make it available using the `virus_sequence.submitter_id` as well. Because virus sequencing is raw sequencing and VCF is more like processed/analysis-ready data.  And it is a complete bioinformatic workflow from virus sequence ---> vcf.  Therfore i'd like to keep the link in between.
Please see the slack thread here: https://cdis.slack.com/archives/CV4HHPB9Q/p1602600381094600